software/twenty.yml

```diff
@@ -0,0 +1,11 @@
+name: Twenty
+website_url: "https://twenty.com"
+source_code_url: "https://github.com/twentyhq/twenty"
+description: A modern CRM offering the flexibility of open source, advanced features, and a sleek design.
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+tags:
+  - Groupware
+demo_url: "https://app.twenty.com"
```

Comment:
```suggestion
```

The demo requires creating an account, then asks you to sign up for a paid plan

![image](https://github.com/awesome-selfhosted/awesome-selfhosted-data/assets/3950390/f3bb0a7b-7032-4c1c-b2e7-d04cb7672127)

Unless there is a way to experiment with the software without signing up or paying, please remove the demo link

Reply:
The demo url has been removed. Thank you! @nodiscc